### DISCUSSION TITLE
Axios and Fetch API error [ OBJECT PROMISE ]

### DISCUSSION THREAD

**Discussion by bobwatcherx** [NativeScript/NativeScript#9740] [Q&A] [answered]
### Issue Description

cannot fetch API 

### Reproduction

i have problem for Fetch data from API

this my code
**Home.svelte**

```
<script >
    import {onMount} from 'svelte'
    let nama = "jono";
onMount(()=>{
     fetch("https://httpbin.org/get").then((res)=>console.log(res.json()))
    .catch((err)=>console.log(err))
})
</script>
<page actionBarHidden="true">
 .....other code
</page>
```
**My Console**
To start debugging, open the following URL in Chrome:
devtools://devtools/bundled/inspector.html?ws=localhost:40000

Successfully synced application org.nativescript.sveltenative on device 49518a86.
JS: [object Promise]

can you have solution ? thanks 

### Environment

env:
svelte-native

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by rigor789** [comment, marked as answer]
See https://developer.mozilla.org/en-US/docs/Web/API/Response/json

`.json()` returns a Promise, so the output is correct. You need to wait for it to resolve:

```js
fetch("https://httpbin.org/get")
  .then(res => res.json())
  .then(res => console.log(res))
  .catch(err => console.log('err', err))
```
